tsconfig.json

```diff
@@ -12,6 +12,7 @@
     "isolatedModules": true,
     "jsx": "preserve",
     "incremental": true,
+    "baseUrl": "src",
```

Comment:
structure is bit confusing Hayk jan, why we created /src folder separately  ? we already have default app directory, 

so rest of the folders can bee on same level

also, I see you created pages/_app.tsx, so this structure is kind of mix of `App` and `Pages` directories of NextJS, which sounds strange, let's please re-think this approach